includes/fbproduct.php

```diff
@@ -447,6 +464,62 @@ public function get_fb_description() {
 		return apply_filters( 'facebook_for_woocommerce_fb_product_description', $description, $this->id );
 	}
 
+	public function get_rich_text_description() {
+		$rich_text_description = '';
+
+		if ( $this->fb_description ) {
```

Comment:
**Code Commenting Best Practices**
When writing code, it's essential to include comments that explain the logic behind our decisions. This is especially important when determining the preferred product description, as advertisers often rely on code comments to understand the behavior of each product attribute.
Something like:
// Determine the preferred product description based on the following logic:
// 1. Check if the rich text description is available and not empty.
// 2. If the rich text description is available, use it as the preferred description.
// 3. Otherwise, fall back to the plain text description.

By adding comments like this, we can make our code more readable and maintainable, and provide better documentation for advertisers who rely on our code comments to understand the behavior of our product attributes.